src/rendering/vdom.ts

```diff
@@ -12,17 +12,28 @@ export interface VNode {
   key: any
 }
 
+// Expose to allow debugging of keys on line elements
+export const options = {
+  renderKeys: false,
+};
+
```

Comment:
A global option to display `data-key="xyz"` on line elements for debugging and testing.